packages/create-app/src/lib/utils/prompts.ts

```diff
@@ -158,19 +160,39 @@ export function promptBundlers(
   });
 }
 
-export function promptRemoteCacheProvider(): Promise<SupportedRemoteCacheProviders | null> {
+export function promptRemoteCacheProvider(
+  providers: RemoteCacheTemplateInfo[],
+): Promise<RemoteCacheTemplateInfo | null> {
   return promptSelect({
     message: 'Which remote cache provider do you want to use?',
-    initialValue: 'github-actions',
-    options: [
-      {
-        value: 'github-actions',
-        label: 'GitHub Actions',
-        hint: 'Enable builds on your CI',
-      },
-      { value: null, label: 'None', hint: 'Local builds only' },
-    ],
-  }) as Promise<SupportedRemoteCacheProviders | null>;
+    initialValue: providers[0],
+    options: providers.map((provider) => ({
+      value: provider,
+      label: provider.displayName,
+    })),
+  });
+}
+
+export function promptRemoteCacheProvidersConfig(
+  provider: SupportedRemoteCacheProviders,
+) {
+  switch (provider) {
+    case 'github-actions':
+      return promptGroup({
+        owner: () => promptText({ message: 'GitHub owner' }),
+        repo: () => promptText({ message: 'GitHub repo' }),
+        token: () =>
+          promptPassword({ message: 'GitHub Personal Access Token (PAT)' }),
+      });
+    case 's3':
+      return promptGroup({
+        bucket: () => promptText({ message: 'S3 bucket' }),
+        region: () => promptText({ message: 'S3 region' }),
+        accessKeyId: () => promptText({ message: 'S3 access key ID' }),
+        secretAccessKey: () =>
+          promptPassword({ message: 'S3 secret access key' }),
```

Comment:
Let's remove these, and tell user to use env variables (link to docs) that will hold it

Reply:
https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-envvars.html#envvars-list?

Reply:
or https://docs.aws.amazon.com/sdkref/latest/guide/feature-static-credentials.html maybe? technically a region could also be parametrized the same way

Reply:
I removed both, but will just need to add an info banner before prompting for S3 args